site/_posts/2017-10-20-hawkular-alerts-in-miq.md

```diff
@@ -7,6 +7,9 @@ published: true
 tags: tutorials middleware
 ---
```

Comment:
Can you please remove the middleware tag? We're keeping the blog categories pretty broad for now, and this is the only post in the middleware category, and there probably won't be many more after this. Thanks!

Reply:
It's removed now.